### ISSUE TITLE
Invalid format error 

### ISSUE THREAD

**Issue by jawndiego** [entropyxyz/entropy-core#847]
in the ``UserSignatureRequest `` struct: 

``` rust
pub struct UserSignatureRequest {
    /// Hex-encoded raw data to be signed (eg. hex-encoded RLP-serialized Ethereum transaction)
    pub message: String,
    /// Hex-encoded auxilary data for program evaluation, will not be signed (eg. zero-knowledge proof, serialized struct, etc)
    pub auxilary_data: Option<Vec<Option<String>>>,
    /// Information from the validators in signing party
    pub validators_info: Vec<ValidatorInfo>,
    /// When the message was created and signed
    pub timestamp: SystemTime,
    /// Hashing algorithm to be used for signing
    pub hash: HashingAlgorithm,
    /// The veryfying key for the signature requested
    pub signature_verifying_key: Vec<u8>,
}```

``pub signature_verifying_key`` is expected to be a unit8array. However, we cant send a unit8array via the ``sign_tx`` RPC method. 

the current issue is that when we pass signature_verifying_key as a string we get: 

        ```
   const txRequestData: UserSignatureRequest = {
          message: stripHexPrefix(strippedsigRequestHash),
          auxiliaryData,
          validatorsInfo: validatorsInfo,
          timestamp: this.getTimeStamp(),
          hash,
          signature_verifying_key: signatureVerifyingKey,
        }```

``  Error: request failed 500, Internal Server Error fetch: http://alice-tss-server:3001/user/sign_tx FULLRESPONSE: Serde Json error: invalid type: string "0x0322931bd57e7507336c58189d41738b5d14ac398c8679b83ecc2437c48af51261", expected a sequence at line 1 column 1042``

but when we pass it as a unit8array: we get:

```   
const txRequestData: UserSignatureRequest = {
          message: stripHexPrefix(strippedsigRequestHash),
          auxiliaryData,
          validatorsInfo: validatorsInfo,
          timestamp: this.getTimeStamp(),
          hash,
          signature_verifying_key: hexToU8a(signatureVerifyingKey),
        }
```

``  ✖ Error: request failed 500, Internal Server Error fetch: http://alice-tss-server:3001/user/sign_tx FULLRESPONSE: Serde Json error: invalid type: map, expected a sequence at line 1 column 972``

**Comment by ameba23** [comment]
### TLDR: I think the verifying key should be given as a JS [`Array`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array)

Serde-JSON appears to interpret `Vec<u8>` as a JSON Array of unsigned 8 bit numbers, eg: `[1,2,3,4]`.  Not a javascript `Uint8Array` or any other type.

Here is an example JSON-serialized `UserSignatureRequest` which i logged out from our tests:
```json
{"message":"536f6d65206d657373616765206d6f7265207468616e203130206279746573","auxilary_data":[null],"validators_info":[{"x25519_public_key":[161,108,23,168,165,13,123,93,229,3,251,13,77,29,68,78,48,39,246,122,221,224,242,134,9,158,89,13,155,66,90,41],"ip_address":"44.193.219.232:3001","tss_account":"5CJinnR8PLjbTgbCWYBrhGLoUUn8Ljp5HY4KrCNLKmqMP5P3"},{"x25519_public_key":[206,209,199,111,242,170,128,120,10,150,100,23,141,250,223,76,217,145,164,52,123,247,140,240,158,34,230,167,72,40,93,28],"ip_address":"52.205.187.95:3001","tss_account":"5CaQHb7jsK1zUAPbnMqsGVDoxm6vNn5DTATTmxnzvSCSYyKk"}],"timestamp":{"secs_since_epoch":1716274316,"nanos_since_epoch":27701089},"hash":"keccak","signature_verifying_key":[3,196,134,154,70,67,166,225,212,8,235,124,100,40,95,197,6,78,161,125,29,146,66,128,247,184,247,46,42,134,112,62,173]}
```

I'm not sure what type your `signatureVerifyingKey` is, but to convert a JS `Uint8Array` or a NodeJS `Buffer` to a 'normal' JS `Array` you can use [`Array.from`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from):

Eg with the NodeJS repl:
```
> Array.from(new Uint8Array(Buffer.from('sdfds')))
[ 115, 100, 102, 100, 115 ]
> JSON.stringify(Array.from(new Uint8Array(Buffer.from('sdfds'))))
'[115,100,102,100,115]'
>
```

**Comment by jawndiego** [comment]
thank you 

``          signature_verifying_key: Array.from(Buffer.from(hexSig, 'hex')),
``

this seemed to work!
